### ISSUE TITLE
Ignore has no 'id' member (no-member)

### ISSUE THREAD

**Issue by a-da** [OCA/pylint-odoo#120]
How can I ignore  `has no 'id' member` ?

	************* Module my_module.contact
	W:100, 0: TODO: rename this field (fixme)	
	C: 46, 0: Line too long (102/100) (line-too-long)
	C: 63, 0: Line too long (108/100) (line-too-long)
	C: 70, 0: Line too long (104/100) (line-too-long)
	C: 72, 0: Line too long (114/100) (line-too-long)
	C: 15, 0: Invalid constant name "_logger" (invalid-name)
	E:197,19: Instance of 'Contact' has no 'id' member (no-member) <<<<<
	E:223,55: Instance of 'Contact' has no 'id' member (no-member) <<<<

The odoo code looks like:

     class Contact(models.Model):
          ......
          if self.id in agreement_ids:

**Comment by naglis** [comment]
I don't think this issue is related to _pylint-odoo_.

You can disable it locally, by adding a comment, eg.:
```python
 class Contact(models.Model):
      ......
      if self.id in agreement_ids:  # pylint: disable=no-member
```

Or you can disable it via command line argument:
```sh
pylint --disable=no-member ...
```

Or disable it in your _.pylintrc_ file:
```ini
[MASTER]
disable = no-member 
```

Please refer to [pylint documentation](https://pylint.readthedocs.io/en/latest/user_guide/run.html) for more information.

**Comment by a-da** [comment]
@naglis, It is related to odoo, `models.Model.id` should is inherited in all model classes.

> You can disable it locally,

If I have 100 `self.id` usages, I should add 100 comments. I am not happy about this.

> Or you can disable it via command line argument, Or disable it in your .pylintrc file:

I think this will disable ALL members checking. It will blind completely linter.

Actually, the check is very simple.
If there is `_name` or `_inherit/inherits` attribute in class then disable `self.id` warning, only and only for that model context/scope.

This can be implemented as separate `pylint-odoo-filter` addons, but I think this should be implemented inside pylint-odoo, because only odoo users need this.

**Comment by moylop260** [comment]
The check `no-member` is not a pylint-odoo custom check.
You should create a issue from the original project: https://github.com/pycqa/pylint/issues/new where the check is created.

(Don't forget send the version and other important information of the template: https://github.com/PyCQA/pylint/blob/master/.github/ISSUE_TEMPLATE.md)
